packages/frontend/src/components/DashboardFilter/FilterConfiguration/TileFilterConfiguration.tsx

```diff
@@ -323,7 +328,37 @@ const TileFilterConfiguration: FC<Props> = ({
             ))}
         </Accordion>
     ) : (
-        <StackSubComponent tileList={tileTargetList} />
+        <Stack spacing="lg">
+            <Checkbox
```

Comment:
Copied from the old code https://github.com/lightdash/lightdash/pull/9884/files#diff-ce5aee5c62719fb1bff56b151b05a1c0806499973b2fe5ef414e52764a686bc5L159